### DISCUSSION TITLE
Multiple locations one address

### DISCUSSION THREAD

**Discussion by JanineKall** [liedekef/events-made-easy#452] [Q&A]
Hi, I've created a map showing multiple locations and a role for users to add their personal address/location. 
Some people might not want to expose their complete address but just fill in #_CITY.  Others might have addresses in the same building, like a tower or an office building. 
If so, pins are plotted over each other and only the newest location is shown. Is it possible to somehow show that multiple pins are underneath?

And / or: if there is no solution to this, is is possible to filter on #_CITY? I created 2 filters with custom fields that work fine, but filtering with #_CITY does not seem to work. 

[https://xyh6t9w.nl/Stage8/](https://xyh6t9w.nl/Stage8/)

**Comment by liedekef** [comment]
What is the shortcode (+ options) you use for the map? Check the option "marker_clustering" if using the shortcode eme_locations_map, see https://www.e-dynamics.be/wordpress/eme_locations_map/
For filtering: it is #_FILTER_CITIES (or #_FILTER_TOWNS) in the filter form format, see https://www.e-dynamics.be/wordpress/277/

**Reply by JanineKall** [reply]
I should have known that you already thought of this. I just overlooked, I'm so sorry. So very happy!
